src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -78,6 +78,8 @@
     public static final String NUM_REDUCERS_LONG_NAME = "num-reducers";
     public static final String SHARDED_OUTPUT_LONG_NAME = "sharded-output";
     public static final String OUTPUT_SHARD_DIR_LONG_NAME = "output-shard-tmp-dir";
+    public static final String WRITE_BAI_LONG_NAME = "write-bai";
+    public static final String WRITE_SBI_LONG_NAME = "write-sbi";
```

Comment:
maybe rename to `create-output-bam-splitting-index` for consistency

Reply:
Done.